### PR TITLE
Starving Demon's Way

### PR DESCRIPTION
**Pull request by vtange**
Poor Lylia

### CODE REVIEW

**Comment by terrapin47** [comment]
:tada: Good to merge

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -2633,10 +2633,10 @@ QUEST_LV_0200_20150317_002632	{memo X}Defeat the monsters in Purple Tree Fault F
 QUEST_LV_0200_20150317_002633	Observe the fallen owl statue
 QUEST_LV_0200_20150317_002634	There is an owl staue in the Tomb Spider nest. Take a look at it.
 QUEST_LV_0200_20150317_002635	Throneweaver appeared when you touched the collapsed statue!
-QUEST_LV_0200_20150317_002636	Suspicous Goddess Statue
-QUEST_LV_0200_20150317_002637	There is a Goddess Statue in Amolallul. This statue looks different from the other statues but worship it first anyway.
-QUEST_LV_0200_20150317_002638	Defeat the Corrupted
-QUEST_LV_0200_20150317_002639	The Goddess Statue turned into Corrupted! Defeat the Corrupted first.
+QUEST_LV_0200_20150317_002636	$Suspicious Goddess Statue
+QUEST_LV_0200_20150317_002637	$There is a Goddess Statue in Amolallul. This statue looks different from the other statues but worship it first anyway.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:2637]: Amolallul Hill

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -3179,44 +3179,44 @@ QUEST_LV_0200_20150323_003178	{memo X}The occurence of Stone Frosts became irreg
...
+QUEST_LV_0200_20150323_003192	$Please think of that necklace as the gift from Lylia and keep it safely.{nl}It is blessed from the great cathedral so it will help you.
+QUEST_LV_0200_20150323_003193	$I am still hungry. Hungry.{nl}I can't take it anymore. I am hungry!
+QUEST_LV_0200_20150323_003194	$I am so.. hungry.{nl}Hey, if you have any leftover food, if you could share it with me..
+QUEST_LV_0200_20150323_003195	$Even if that's all you have, please share it with me.. please..
+QUEST_LV_0200_20150323_003196	$Meat..quickly. {nl}I feel like I am dying.
+QUEST_LV_0200_20150323_003197	$Meat.. Meat!{nl}Please give it to me fast. Fast!
+QUEST_LV_0200_20150323_003198	$Wait, that music box.. what was it?{nl}
+QUEST_LV_0200_20150323_003199	$My memories.. she..{nl}Give it to me! That's mine!
+QUEST_LV_0200_20150323_003200	{memo X}Damn. What was I doing?{nl}Ahah. Lylia.. My Lylia!{nl}
+QUEST_LV_0200_20150323_003201	$We left for a pilgrimage to the great cathedral to receive the blessing of the Goddesses to celebrate our wedding.{nl}But then, that evil Naktis.. that unforgivable Demon Lord.{nl}
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:3201]: Great Cathedral?
Goddesses of Goddess? If Goddesses, blessings. If Goddess, blessing

File: QUEST_LV_0200.tsv [hunk 3]
```diff
@@ -4673,46 +4673,46 @@ QUEST_LV_0200_20150323_004672	Defeat the monsters at the Forest of Prayer
...
-QUEST_LV_0200_20150323_004690	Defeat Liliya who turned into a Carnivore
-QUEST_LV_0200_20150323_004691	Make the grave of the pilgrim, Liliya
-QUEST_LV_0200_20150323_004692	Liliya is about to die... Please listen her last words
-QUEST_LV_0200_20150323_004693	Liliya passed away. Please make the grave of Liliya.
+QUEST_LV_0200_20150323_004687	$Did Pilgrim Lylia's hunger go away? Talk to her again
+QUEST_LV_0200_20150323_004688	$Talk to the fallen Pilgrim Lylia
+QUEST_LV_0200_20150323_004689	$Pilgrim Lylia appeared again from the place where the Carnivore was defeated. Talk to Lylia again.
+QUEST_LV_0200_20150323_004690	$Defeat Lylia who turned into a Carnivore
+QUEST_LV_0200_20150323_004691	$Make a grave for Pilgrim Lylia
+QUEST_LV_0200_20150323_004692	$Lylia is dying... Please listen her last words
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:4692]: To her words

File: QUEST_LV_0200.tsv (hunk 2 above)
Comment (vtange) [inline QUEST_LV_0200.tsv:3201]: will mass replace Great Cathedral in a later commit

**Comment by imcgames** [comment]
:+1:

**Pull request merged**